service/templates/admin.html

```diff
@@ -36,9 +37,14 @@
           </md-card-title-text>
         </md-card-title>
         <md-card-content>
-          Placeholder content
-          <br/><br/><br/>
+          <md-input-container class="control">
+            <label>Site-wide banner:</label>
+            <input type="text" ng-model="Settings.banner" placeholder="(Leave blank to remove banner)">
```

Comment:
does it just make sense to have a "clear message button"

Reply:
I'm not opposed to it, but I like this version better since it keeps things simpler (fewer buttons on the screen since later tickets will add more buttons). Let me know if you disagree though

Reply:
It's just not an idiom we've used in other places (we tend toward the extra button.) and I'm not as compelled by minimalism here. 

if its easy to switch, lets do that.

Reply:
Done